charts/kueue/templates/prometheus/monitor.yaml

```diff
@@ -12,8 +12,7 @@ spec:
     path: /metrics
     port: https
     scheme: https
-    tlsConfig:
```

Comment:
Similarly, I would prefer not to change the default behavior in the PR.

Reply:
This is difficult to decouple as I need to inject custom tlsConfig for this to actually work.

Reply:
We followed off offline on this and we decided to set the defaults the same but we are introducing a config to change tlsConfigs in special cases.